crt_portal/cts_forms/templates/forms/complaint_view/index/complaints_table.html

```diff
@@ -18,6 +19,11 @@
     <tbody>
     {% for datum in data_dict %}
       <tr class="tr-status-{{ datum.report.status }} tr--hover{% cycle ' stripe' '' %}">
+        <td>
+          <a data-id="{{ datum.report.id }}" class="td-toggle display-flex" href="#">
+            <img src="{% static "img/intake-icons/ic_chevron-right.svg" %}" alt="toggle" class="icon">
```

Comment:
I think there's an opportunity here to provide a more descriptive aria-label for our friends using screen readers

Reply:
Right you are! Added more explanation in the `alt` attribute. Thanks.